blog_post.md

```diff
@@ -0,0 +1,144 @@
+### GSoC 2016 Summary, Adding categorical data support
+
+Support for categorical data is important for any data analysis tool. This summer I added categorical data support to:
+- to easily analyze categorical data in Daru
+- visualize categorical data
+- support regression with categorical variable in Statsample and Statsample-GLM
```

Comment:
I think phrasing and grammar in this introductory paragraph needs improvement. Better phrasing and grammar would be something like:

> Support for categorical data is important for any data analysis tool. This summer I implemented categorical data capabilities for:
> - Convenient and efficient data wrangling for categorical data in Daru
> - Visualization of categorical data
> - Multiple linear regression and generalized linear models (GLM) with categorical variables in Statsample and Statsample-GLM
 
Also, make daru, statsample and statsample-glm link to the respective projects.